mobile/src/main/java/ch/epfl/sdp/mobile/state/StatefulHome.kt

```diff
@@ -17,6 +17,9 @@ private const val SocialRoute = "social"
 /** The route associated to the settings tab. */
 private const val SettingsRoute = "settings"
 
+/** The route associated to editing button on the setting screen. */
+private const val SettingEditProfileNameRoute = "/profile-name/edit"
```

Comment:
I'd try to keep the path consistent with the other routes.

```suggestion
private const val SettingEditProfileNameRoute = "profile-name/edit"
```